curriculum/structure/blocks/workshop-string-inspector.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Build a String Inspector",
-  "isUpcomingChange": true,
+  "isUpcomingChange": false,
```

Comment:
Just to be clear, we are releasing this block?

Reply:
Yes, that should be released 👍🏾